### ISSUE TITLE
Enhancement: Release for macOS (sans playback, if incompatible)

### ISSUE THREAD

**Issue by mhuusko5** [spotify/ios-sdk#703]
Title says it. I'm guessing the SDK could be compiled/distributed for macOS nearly as-is (even the playback bits, but who knows, maybe some iOS specific stuff is going on under the hood along with the portable CoreAudio functionality), and that's a colossal value add.

If this isn't going to happen, would be good to know so those hoping to deploy to macOS/share code can start building alternate SDK/wrappers around the Web API (without playback, unfortunately).

**Comment by loretoparisi** [comment]
💯

**Comment by Anders27** [comment]
Yes, please give as a Mac OS X SDK, sorry macOS SDK. 99% of the work should already be done since iOS and MacOS is practically the same in terms of a Spotify SDK.

And please, also a Windows SDK!

**Comment by DmytroAnkudinov** [comment]
Added "playback" label as this is currently the only framework where enhancement would matter (in terms of this repository issue list) in the future.
